lib/modules/datasource/bitbucket-tags/index.ts

```diff
@@ -1,6 +1,7 @@
 import { cache } from '../../../util/cache/package/decorator';
 import { BitbucketHttp } from '../../../util/http/bitbucket';
 import { ensureTrailingSlash } from '../../../util/url';
+import type { PagedResult, RepoInfoBody } from '../../platform/bitbucket/types';
```

Comment:
don't make changes to unrelated code, open separate PR's if needed

Reply:
Sure, I can move this to a separate PR if you really prefer.  I figured after adding a dozen or so new types for Jira Cloud that this would also make sense together.

Reply:
ref https://github.com/renovatebot/renovate/pull/20570